blockmanager.go

```diff
@@ -452,9 +447,8 @@ func (b *blockManager) handleTxMsg(tmsg *txMsg) {
 
 	// Process the transaction to include validation, insertion in the
 	// memory pool, orphan handling, etc.
-	allowOrphans := cfg.MaxOrphanTxs > 0
 	acceptedTxs, err := b.txMemPool.ProcessTransaction(tmsg.tx,
-		allowOrphans, true, mempool.Tag(tmsg.peer.ID()))
```

Comment:
It's not a big deal, but I'm not sure I agree here.  The intention of a reject message is to inform the remote peer that you rejected the transaction.  It's true that nothing is really done about that by an peers on the network right now, but it would be possible to notice that your txns are consistently being rejected by a peer and decide to disconnect and go find greener pastures.